src/pynwb/ndx_photometry/fibers_table.py

```diff
@@ -6,52 +6,55 @@
 from pynwb.core import VectorIndex
 
 
-@docval({'name': 'excitation_source',
-         'type': int,
-         'doc': 'references rows of ExcitationSourcesTable',
-         'default': None},
-        {'name': 'photodetector',
-         'type': int,
-         'doc': 'references rows of PhotodetectorsTable',
-         'default': None},
-        {'name': 'fluorophores',
-         'doc': 'references rows of FluorophoresTable',
-         'type': 'array_data',
-         'default': None,
-         'shape':(None,)},
-        {'name': 'location',
-         'type': str,
-         'doc': 'location of fiber',
-         'default': None},
-        {'name': 'notes',
-         'type': str,
-         'doc': 'description of fiber',
-         'default': None},
-        {'name': 'fiber_model_number',
-         'type': str,
-         'doc': 'fiber model number',
-         'default': None},
-        {'name': 'dichroic_model_number',
-         'type': str,
-         'doc': 'dichroic model number',
-         'default': None},
-        allow_extra=True)
+@docval(
+    {
+        'name': 'photodetector',
+        'type': int,
+        'doc': 'references rows of PhotodetectorsTable',
+        'default': None
+    },
+    {
+        'name': 'location',
+        'type': str,
+        'doc': 'location of fiber',
+        'default': None
+    },
+    {
+        'name': 'notes',
+        'type': str,
+        'doc': 'description of fiber',
+        'default': None
+    },
+    {
+        'name': 'fiber_model_number',
+        'type': str,
+        'doc': 'fiber model number',
+        'default': None
+    },
+    {
+        'name': 'dichroic_model_number',
+        'type': str,
+        'doc': 'dichroic model number',
+        'default': None
+    },
+    allow_extra=True
+)
 def add_fiber(self, **kwargs):
     """
     Add a row per fiber to the fibers table
     Checks to see if the tables are properly referenced
     If not, gets their references from the nwbfile
     """
     super(FibersTable, self).add_row(**kwargs)
-    referenced_tables = ('excitation_sources','photodetectors','fluorophores')
+    referenced_tables = ('photodetectors', )
```

Comment:
Shouldn't this be removed as well?
https://github.com/catalystneuro/ndx-photometry/pull/7#issuecomment-1989704848

Reply:
Hmm then maybe there's no need for `fibers_table.py` either?